src/ocm/components/clusterConfiguration/networkConfiguration/StackTypeControl.tsx

```diff
@@ -90,9 +90,12 @@ export const StackTypeControlGroup = ({
 
   const setDualStack = () => {
     setFieldValue('stackType', DUAL_STACK);
-    setFieldValue('networkType', getDefaultNetworkType(isSNO, true));
     setFieldValue('vipDhcpAllocation', false);
 
+    if (values.networkType === NETWORK_TYPE_SDN && !canSelectNetworkTypeSDN(isSNO, true)) {
```

Comment:
Isn't the `&& !canSelectNetworkTypeSDN(isSNO, true)` part unnecessary?

Reply:
With the current rules, yes it's unnecessary.

But I think this way is better because the code is being explicit as to why it's modifying the networkType, and also because if the rules ever change of when SDN is a valid option, then the only change needed is in `canSelectNetworkTypeSDN`.

Anyway, it can be removed so whatever you feel is better.

Reply:
The answer to the last question is no, because SDN was selectable for a previous status of the cluster (its stackType, and whether or not it's an SNO). And now we're checking if the clusters' new status (eg a change in its stackType) lets us keep the existing selection.

This is what the second part of the IFs tries to be clear about, that we're only updating networkType because now its value becomes invalid.

As I mentioned it can be removed, so it may be the best option to do so.

Reply:
How is it relevant that SDN was selectable in the previous state if we the networking type value is currently SDN and we are changing it to OVN anyway?

Reply:
Anyway, I was just trying to understand if there was a good reason to keep it there. But I do think it's best if this part of the condition is removed.

Reply:
I understand your point, and I'll remove it